### PR TITLE
Add (AnimeXNovel) presence

### PR DESCRIPTION
**Pull request by Nanashinn**
Proof of working:
https://imgur.com/a/dtHWZoj

### CODE REVIEW

**Comment by Nanashinn** [comment]
image
image
more proofs

File: AnimeXNovel/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "author": {
+      "name": "Nanashin",
+      "id": "330030557457022986"
+    },
+    "service": "AnimeXNovel",
+    "description": {
+      "pt_BR": "Leia novels e mangás."
+    },
```
Comment (Bas950) [inline AnimeXNovel/dist/metadata.json:9]: you need an English description

File: AnimeXNovel/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "name": "Nanashin",
+      "id": "330030557457022986"
+    },
+    "service": "AnimeXNovel",
+    "description": {
+      "pt_BR": "Leia novels e mangás."
+    },
+    "url": "www.animexnovel.com",
+    "version": "1.0.0",
+    "logo": "https://2.bp.blogspot.com/-xOR5ZIeXSUk/XdSQBjbtbOI/AAAAAAAANWE/GdwZIwTb3sw5GbWiHp6vys59QNoaM8D6QCK4BGAYYCw/s274/logo.jpg",
```
Comment (Bas950) [inline AnimeXNovel/dist/metadata.json:12]: upload to imgur

File: AnimeXNovel/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "thumbnail": "https://imgur.com/a/XUM8Pmz",
+    "color": "#b4ad9d",
```
Comment (Bas950) [inline AnimeXNovel/dist/metadata.json:14]: wrong imgur link, should be i.imgur link ending with .png/.jpg/etc.

File: AnimeXNovel/dist/presence.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
...WUFDTCxRQUFRLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyx3QkFBd0IsQ0FBQyxDQUFDLFdBQVcsQ0FBQztZQUN4RSxJQUFJLENBQUMsT0FBTyxHQUFHLGlCQUFpQixDQUFDO1lBQ2pDLElBQUksQ0FBQyxLQUFLLEdBQUcsUUFBUSxDQUFDO1lBQ3RCLElBQUksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1NBQ3JDO0tBQ0Y7U0FBTSxJQUFJLFFBQVEsQ0FBQyxRQUFRLENBQUMsUUFBUSxDQUFDLFVBQVUsQ0FBQyxRQUFRLENBQUMsRUFBRTtRQUMxRCxJQUFJLFFBQVEsQ0FBQyxRQUFRLENBQUMsUUFBUSxDQUFDLFFBQVEsQ0FBQyxTQUFTLENBQUMsRUFBRTtZQUNsRCxJQUFJLEtBQUssR0FBRyxRQUFRLENBQUMsYUFBYSxDQUFDLDZCQUE2QixDQUFDO2lCQUM5RCxXQUFXLENBQUM7WUFDZixJQUFJLENBQUMsT0FBTyxHQUFHLGlCQUFpQixDQUFDO1lBQ2pDLElBQUksQ0FBQyxLQUFLLEdBQUcsS0FBSyxDQUFDO1lBQ25CLElBQUksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1NBQ3JDO2FBQU07WUFDTCxJQUFJLENBQUMsT0FBTyxHQUFHLG9CQUFvQixDQUFDO1lBQ3BDLElBQUksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1NBQ3JDO0tBQ0Y7U0FBTSxJQUFJLFFBQVEsQ0FBQyxRQUFRLENBQUMsUUFBUSxDQUFDLFVBQVUsQ0FBQyxTQUFTLENBQUMsRUFBRTtRQUMzRCxJQUFJLE1BQU0sR0FBRyxRQUFRLENBQUMsYUFBYSxDQUFDLDZCQUE2QixDQUFDO2FBQy9ELFdBQVcsQ0FBQztRQUNmLElBQUksQ0FBQyxPQUFPLEdBQUcsa0JBQWtCLENBQUM7UUFDbEMsSUFBSSxDQUFDLEtBQUssR0FBRyxNQUFNLENBQUM7UUFDcEIsSUFBSSxDQUFDLGNBQWMsR0FBRyxhQUFhLENBQUM7S0FDckM7SUFDRCxRQUFRLENBQUMsV0FBVyxDQUFDLElBQUksQ0FBQyxDQUFDO0FBQzdCLENBQUMsQ0FBQyxDQUFDIn0=
```
Comment (Bas950) [inline AnimeXNovel/dist/presence.js:62]: You cant have sourcemapping if you only have a js file, so upload the ts files or remove it.
I suggest you to upload the ts files

Comment (Nanashinn) [reply AnimeXNovel/dist/presence.js]: I removed the sourcemapping, because i don't know TS

**Pull request merged**